server/tests/controllers/miners/github/test_pull_request.py

```diff
@@ -2017,3 +2018,95 @@ async def test_mine_pull_requests_event_releases(
     facts2 = await calc(*args)
     facts2.sort_values(PullRequestFacts.f.created, inplace=True, ignore_index=True)
     assert_frame_equal(facts1.iloc[1:], facts2.iloc[1:])
+
+
+def test_pull_request_review_request_without_review_open():
+    miner = PullRequestFactsMiner(set())
+    pr = MinedPullRequest(
```

Comment:
@gaetano-guerriero do we already have a factory for this? It would be really handy...